nss-cohort-40/bangazon-ecommerce-client-dead-parrots-frontend
# Homepage rl

To test:
Login or Register
Should see products listed on homepage if you have some in your db
Click on the product and you should be directed to the product details
